.circleci/config.yml

```diff
@@ -125,8 +125,8 @@ jobs:
     environment:
       CIRCLE_TEST_REPORTS: /home/circleci/junit
       CIRCLE_ARTIFACTS: /home/circleci/artifacts
-      GRADLE_OPTS: -Dorg.gradle.jvmargs='-XX:MaxMetaspaceSize=256m' -Dorg.gradle.workers.max=2
```

Comment:
this will get overwritten by the circle template... is there a number we could dial this up to that would make this stuff pass ? alternatively would the forkEvery option just spin up more JVMs?

Reply:
Not if the circle template can't run tests ;-)
Agree this isn't the long term solution, I just wanted the tests to run without hitting resource limits.